system-tests/lib/cre/don/gateway/gateway.go

```diff
@@ -39,7 +39,6 @@ func CreateJobs(ctx context.Context, creEnv *cre.Environment, dons *cre.Dons, ga
 			JobName:     "gateway-worker",
 			DONFilters: []offchain.TargetDONFilter{
 				{Key: offchain.FilterKeyDONName, Value: gatewayNode.DON.Name},
-				{Key: "p2p_id", Value: gatewayNode.Keys.PeerID()},
```

Comment:
Removed because system-tests don't set register to JD with this label. A better fix would be to actually add the labels, but for time this will be handled in a separate ticket (CRE-1305).